applications/ecallmgr/src/ecallmgr_call_events.erl

```diff
@@ -998,6 +998,13 @@ get_disposition(Props) ->
 get_hangup_code(Props) ->
     kzd_freeswitch:hangup_code(Props).
 
+-spec get_billing_seconds(wh_proplist()) -> api_binary().
+get_billing_seconds(Props) ->
+    case props:get_integer_value(<<"variable_billmsec">>, Props) of
+        'undefined' -> props:get_value(<<"variable_billsec">>, Props);
+        Billmsec -> wh_util:to_binary(wh_util:ceiling(Billmsec / 1000))
```

Comment:
We can always add a second to `variable_billsec` but I thought since we don;t have enough time resolution it's better leave seconds as is(what if it's round exactly, e.g. 4). If someone needs more precise billing based on millisecond then he can enable `variable_billmsec` on FreeSWITCH.